src/config/config-alpha.js

```diff
@@ -9,6 +9,7 @@ export const config = {
    * API Calls Related
    */
   appEnv: 'prod', // helps in deciding some text, links, etc, DO NOT CHANGE
+  actualEnv: 'alpha', // helps in deciding some text, links, etc, DO NOT CHANGE
```

Comment:
The changes done in config files aren't related to this PR. Can we do it in a separate PR.

I am also not quite sure what how these changes will affect the code. So I will appreciate if we could have a separate PR with proper details in the ticket description about these changes :)